wolfssl/wolfcrypt/ed448.h

```diff
@@ -102,10 +104,27 @@ int wc_ed448ph_sign_hash(const byte* hash, word32 hashLen, byte* out,
                          word32 *outLen, ed448_key* key,
                          const byte* context, byte contextLen);
 WOLFSSL_API
+int wc_ed448_sign_msg_ex(const byte* in, word32 inLen, byte* out,
+                         word32 *outLen, ed448_key* key, byte type,
+                         const byte* context, byte contextLen);
+WOLFSSL_API
 int wc_ed448ph_sign_msg(const byte* in, word32 inLen, byte* out,
                         word32 *outLen, ed448_key* key, const byte* context,
                         byte contextLen);
 WOLFSSL_API
+int wc_ed448_verify_msg_ex(const byte* sig, word32 sigLen, const byte* msg,
+                            word32 msgLen, int* res, ed448_key* key,
+                            byte type, const byte* context, byte contextLen);
+WOLFSSL_API
+int wc_ed448_verify_msg_init(const byte* sig, word32 sigLen, ed448_key* key,
+                        byte type, const byte* context, byte contextLen);
+WOLFSSL_API
+int wc_ed448_verify_msg_feed(const byte* msgSegment, word32 msgSegmentLen,
```

Comment:
Change to wc_ed448_verify_msg_update

Reply:
works for me.  will do that for `wc_ed25519_verify_msg_update()` too.  but on Tuesday -- holiday today :grin: